e2eshark/tools/stubs/commonutils.py

```diff
@@ -43,22 +43,31 @@ def flatten_tuples(tup):
     return test_op_list
 
 
+# listOfFunctionArgRetTuple = (function, [args other than input], isReturnTuple, indexOfTupleForTupleReturn)
+def applyPostProcessPipeline(item, functionPipeLine):
```

Comment:
Comment doesn't match the function parameter name? Should this be 'functionPipeLine'?

You could also use a named tuple or class instead of an unstructured tuple: https://docs.python.org/3/library/collections.html#collections.namedtuple